shell/platform/common/accessibility_bridge.h

```diff
@@ -106,6 +109,33 @@ class AccessibilityBridge
   const std::vector<ui::AXEventGenerator::TargetedEvent> GetPendingEvents()
       const;
 
+  // AXTreeManager methods.
+
+  ui::AXNode* GetNodeFromTree(const ui::AXTreeID tree_id,
```

Comment:
It seems the style guide is to label each method with `// |AXTreeManager|` to declare which type these methods implement.